portfolio_lu.html

```diff
@@ -0,0 +1,81 @@
+          </tr>
+          <tr>
+            <td><h4 style="color:whitesmoke; text-align:center">Si longus, levis. Idemne, quod iucunde? Negat enim summo bono afferre incrementum diem.</h4></td>
+            <td><h4 style="color:whitesmoke; text-align:center">Si longus, levis. Idemne, quod iucunde? Negat enim summo bono afferre incrementum diem.</h4></td>
+            <td><h4 style="color:whitesmoke; text-align:center">Si longus, levis. Idemne, quod iucunde? Negat enim summo bono afferre incrementum diem.</h4></td>
+          </tr>
+        </table>
+        </section>
+        <section style="background-color:gold">
+          <h1 id="meus_trabalhos" style="color:deeppink; text-align:center">Meus Trabalhos</h1>
+          <h3 style="color:black; text-align:center">Dois mollius placerat quellen ergo est bloo dignissum canis</h3>
+          <ul>
+             <li>Tudo</li>
+             <li>Website</li>
+             <li>Template</li>
+             <li>Tema</li>
+          </ul>
+          <img src="https://github.com/lucominato/front-end-study-group-rj/blob/master/images/Portfolio01.png?raw=true" width="290" height="290">
+          <img src="https://github.com/lucominato/front-end-study-group-rj/blob/master/images/Portfolio02.png?raw=true" width="290" height="290">
+          <img src="https://github.com/lucominato/front-end-study-group-rj/blob/master/images/Portfolio03.png?raw=true" width="290" height="290">
+        </section>
+        <section style="background-color:black">
+          <h1 id="quem_sou_eu" style="color:deeppink; text-align:center">Quem Sou Eu</h1>
+          <h3 style="color:orangered; text-align:center">Dois mollius placerat quellen ergo est bloo dignissum felinum</h3>
+          <img src="https://github.com/lucominato/front-end-study-group-rj/blob/master/images/photo.png?raw=true" width="290" height="290"> 
+          <h4 style="color:orangered; text-align:center">Joana d'Arc</h4>
+        <p><h2 style="color:orangered; text-align:center">Sobre mim</h2></p> 
+          <h4 style="color:orangered; text-align:center">Lorem ipsum dolor sit amet, consectetur adipiscing elit. His similes sunt omnes, qui virtuti student levantur vitiis, levantur erroribus, nisi forte censes Ti. Re mihi non aeque satisfacit, et quidem locis pluribus. Quippe: habes enim a rhetoribus; Duo Reges: constructio interrete.</h4> 
+          <h3 style="color:orangered">Habilidades</h3>
+          <h4 style="color:whitesmoke">JavaScript</h4>   
+          <h4 style="color:whitesmoke">HTML & CSS</h4>
+          <h4 style="color:whitesmoke">jQuery</h4>
+          <h4 style="color:whitesmoke">Wordpress</h4>   
+        </section>
+        <section style="background-color:gold">
+          <h1 id="contato" style="color:deeppink; text-align:center">Contato</h1>
+          <h3 style="color:black; text-align:center">Dois mollius placerat quellen ergo est sancto</h3>
+          <h3 style="color:black; text-align:center">mail@email.com</h3>

```

Comment:
```suggestion
          <p>mail@email.com</p>
          <p>+55 21 23456-7890</p>
```

O mesmo vale para esses dois :)